### DISCUSSION TITLE
Migration existing Trello boards into Github projects

### DISCUSSION THREAD

**Discussion by abhik2903** [community/community#41179] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

i am planning to migrate our existing trello boards to github projects but no idea how to do it. If anyone has done this before or has idea kindly let me know.

Thanks in advancs

**Comment by bhavaniravi** [comment]
I found this repo on github, the app seems to be down but you can run it locally and test it out
https://github.com/okcoker/TrelloToGithubProjects

**Reply by abhik2903** [reply]
thanks @bhavaniravi i will check it out
